PyInstaller/hooks/rthooks/pyi_rth_inspect.py

```diff
@@ -14,15 +14,52 @@ def _pyi_rthook():
     import inspect
     import os
     import sys
+    import zipfile
 
     # Use sys._MEIPASS with normalized path component separator. This is necessary on some platforms (i.e., msys2/mingw
     # python on Windows), because we use string comparisons on the paths.
     SYS_PREFIX = os.path.normpath(sys._MEIPASS)
+    BASE_LIBRARY = os.path.join(SYS_PREFIX, "base_library.zip")
 
-    _orig_inspect_getsourcefile = inspect.getsourcefile
+    # Obtain the list of modules in base_library.zip, so we can use it in our `_pyi_getsourcefile` implementation.
+    def _get_base_library_files(filename):
+        # base_library.zip might not exit
+        if not os.path.isfile(filename):
+            return {}
+
+        with zipfile.ZipFile(filename, 'r') as zf:
+            namelist = zf.namelist()
+
+        return set(os.path.normpath(entry) for entry in namelist)
```

Comment:
Should that `return {}` be `return set()`? It looks like the output is only used for `in` operations so I guess it wouldn't matter but it's weird seeing the mismatching return types.

Reply:
Yeah, it shoud be a `return set()`. Thanks!